.github/workflows/python-app.yml

```diff
@@ -31,11 +32,14 @@ jobs:
         cache: 'poetry'
 
     - name: Install dependencies
-      run: poetry install
+      run: poetry install --no-interaction --with dev --extras all_plugins
 
     - name: Lint formatting
       run: poetry run black --check .
 
     - name: Test with pytest
       run: poetry run pytest
 
+    - name: Run mypy
```

Comment:
As long as this doesn't cause problems with modelbench, I'm fine with it. Otherwise, let's restrict it to modelgauge.

Reply:
What kind of problems?

Reply:
It was indeed causing problems. I excluded modelbench from this check.